build/commands/lib/config.js

```diff
@@ -132,6 +132,7 @@ const Config = function () {
   this.braveAndroidKeystorePassword = getNPMConfig(['brave_android_keystore_password'])
   this.braveAndroidKeyPassword = getNPMConfig(['brave_android_key_password'])
   this.braveVariationsServerUrl = getNPMConfig(['brave_variations_server_url']) || ''
+  this.dcheckAlwaysOn = getNPMConfig(['dcheck_always_on']) || true
```

Comment:
this is not correct, it will enable dcheck for all builds including release which we definitely do not want

Reply:
The configuration name should specify the default configuration so we should not have to override Release back with an npm config somewhere in CI. You should instead override `dcheck_always_on` to `true` for the specific case of pr builder